manifests/plugin/dynflow/params.pp

```diff
@@ -0,0 +1,8 @@
+# Default parameters for the Dynflow smart proxy plugin
+class foreman_proxy::plugin::dynflow::params {
+  include ::foreman_proxy::params
+
+  $enabled           = true
+  $listen_on         = 'https'
+  $group             = $::foreman_proxy::params::user
```

Comment:
Not sure if this is the best way to go. If the user is overridden, this doesn't pick it up. ```undef``` would since ```foreman_proxy::plugin``` has the correct default but not sure how kafo handles that.